e2e-cypress/files/pdf-generator.ts

```diff
@@ -0,0 +1,68 @@
+import * as puppeteer from 'puppeteer';
+import * as path from 'path';
+import * as fs from 'fs';
+
+async function expandReportTestCases(htmlPage: puppeteer.Page) {
+  await htmlPage.evaluate(() => {
+    const expandTestHeaders = document.querySelectorAll('[class^="test--header"], [class*="test--header"]');
+    for (const testHeader of Array.from(expandTestHeaders)) {
+      (testHeader as HTMLElement).click();
+    }
+  });
+}
+
+const isLocal = process.env.NODE_ENV === 'local';
+
+(async () => {
+  try {
+    const files = await fs.promises.readdir('build/test-results/mochawesome/');
```

Comment:
The foldername 'build/test-results/mochawesome/' is used multiple times. Maybe this should be replaced by a constant or another path variable.